setup.py

```diff
@@ -222,6 +222,7 @@ def _download_and_extract_local_driver(
         "pyee==11.0.1",
         "typing-extensions;python_version<='3.8'",
     ],
+    setup_requires=["setuptools-scm==8.0.4", "wheel==0.41.2"],
```

Comment:
Add a TODO that this should be removed eventually once we use "isolated build system"